tests/test_py2_wsgi.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013 by Rackspace Hosting, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test unicode_literals in python2 invalidating headers which come as Unicode values"""
+from __future__ import unicode_literals
+
+import falcon
+from falcon.testing import TestClient
+
+import pytest
+import six
+
+
+def test_test_client_works_in_py2():
+    app = falcon.API()
+    client = TestClient(app)
+    client.simulate_get('/', headers={
+        'Authorization': 'token blah'
```

Comment:
Another matter of taste thing: instead of ``unicode_literals``, you could just parametrize the test and test for both ``'token_blah'`` and ``u'token_blah'``, which would bring some redundancy on PY3, but something we could probably live with.